File: solutions/javascript/resistor-color-duo/1/resistor-color-duo.js
const COLORS = [
  'black', 'brown', 'red', 'orange', 'yellow',
  'green', 'blue', 'violet', 'grey', 'white'
];

export const decodedValue = ([color1, color2]) => {
  return COLORS.indexOf(color1) * 10 + COLORS.indexOf(color2);
};

File: solutions/julia/resistor-color-duo/1/resistor-color-duo.jl
function colorcode(colors)
    # Словарь сопоставления цветов и их числовых значений
    color_values = Dict(
        "black"  => 0,
        "brown"  => 1,
        "red"    => 2,
        "orange" => 3,
        "yellow" => 4,
        "green"  => 5,
        "blue"   => 6,
        "violet" => 7,
        "grey"   => 8,
        "white"  => 9
    )
    
    # Получаем значения для первых двух цветов
    first_digit = color_values[colors[1]]
    second_digit = color_values[colors[2]]
    
    # Объединяем их в двузначное число
    return first_digit * 10 + second_digit
end
